src/spark_algorithms/README.md

```diff
@@ -0,0 +1,75 @@
+##SYNOPSIS
+This is example code for implementing a word2vec model, term frequency*inverse document frequency (TF-IDF) and others.
+
+##word2vec
+Example of how to use the word2vec model and using a K Means cluster for them.
+
```

Comment:
It was useful in the TFIDF section to know that sentences/documents could be passed in, here the text though is a RDD of Seq[String] (so looks like it does not like sentences - needs everything to be tokenized already potentially).  Adding a bit more to the explanation could help others realize that text variable here and down below in TFIDF are different from one another.